app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,9 @@
+<resources>
+    <string name="search">Suchen</string>
+    <string name="application_not_available">Kein Browser Installiert</string>
+    <string name="custom_provider">Benutzerdefiniert</string>
+    <string name="dialog_title_custom_url">benutzterdefinierter Suchlink</string>
+    <string name="cancel">Abbrechen</string>
+    <string name="select">Auswählen</string>
+    <string name="dialog_hint_custom_url">https://duckduckgo.com?q=%s</string>
```

Comment:
Please remove this line. 
I would need to mark it as not translatable on the default xml but that's something for another commit.

Reply:
By this line I meant 
`<string name="dialog_hint_custom_url">https://duckduckgo.com?q=%s</string>`